pom.xml

```diff
@@ -65,6 +67,9 @@
     <opentelemetry.bom.version>1.38.0</opentelemetry.bom.version>
     <opentelemetry.semconv.version>1.25.0-alpha</opentelemetry.semconv.version>
     <opentelemetry.instrumentation.annotations.version>2.6.0</opentelemetry.instrumentation.annotations.version>
+    <spark.runtime.version>2.4.6</spark.runtime.version>
```

Comment:
can we move these to the spark-hpcc pom? I don't think any other subprojects use these vars